clear-site-data/storage.https.html

```diff
@@ -45,6 +69,18 @@
                 }, test_name);
               });
 
+              promise_test(function() {
+                const serviceWorkerResponseOk = TestUtils.getServiceWorkerResponseOk();
+
+                // Tell the iframe to make a request for the URL that the service worker
+                // responds to
+                serviceWorkerTestPageIFrame.contentWindow.postMessage(null, "*");
+
+                return serviceWorkerResponseOk.then(function(responseIsOk) {
+                  assert_false(responseIsOk, "Response should not be 200");
```

Comment:
I think it would be slightly better to have a network response with a different body available.  That way we could conclusively determine that network was used instead of the service worker.  Currently the failure could indicate the browser is in a broken state, etc.

Reply:
Seems good, added a server endpoint for the request intercepted by the service worker.